contracts/LifCrowdsale.sol

```diff
@@ -257,15 +300,18 @@ contract LifCrowdsale is Ownable, Pausable {
     // foward founds and unpause token only if minCap is reached
     if (funded()) {
 
+      // finish the minting of the token, unpause it and transfer teh ownership to the foundation
       token.finishMinting();
-      forwardFunds();
       token.unpause();
-
       token.transferOwnership(owner);
 
-    }
+      forwardFunds();
 
-    Finalized();
+    } else {
+
+      Finalized();
```

Comment:
I think the event should trigger independently of whether it was successfully funded or not.

Reply:
:+1: